sdk/services/exchange.ts

```diff
@@ -430,7 +430,7 @@ export default class ExchangeService {
 			ethers.utils.formatBytes32String(currencyKey)
 		);
 
-		return numEntries ?? 0;
+		return Number(numEntries.toString()) ?? 0;
```

Comment:
As you replaced numEntries ?? 0 above makes it seems like this can be null or undefined so .toString() could error?

Reply:
Good catch. Replaced with `numEntries ? Number(numEntries.toString()) : 0`.